sprint5/zadanie4.js

```diff
@@ -0,0 +1,22 @@
+/*Return the Last Element in an Array
+Create a function that accepts an array and returns the last item in the array.
+
+Examples
+getLastItem([1, 2, 3]) ➞ 3
+
+getLastItem(["cat", "dog", "duck"]) ➞ "duck"
+
+getLastItem([true, false, true]) ➞ true
+Notes
+Don't forget to return the result.
+If you get stuck on a challenge, find help in the Resources tab.
+If you're really stuck, unlock solutions in the Solutions tab.*/
+
+function getLastItem(arr) {
+  return arr[arr.length -1];
+}
+
+/* Kod tworzy getLastItem tablicę zawierającą 3 elementy. Indeks ma 2.
```

Comment:
Całe to zdanie proszę inaczej sformułować. Zacząć od tego, że `Ja swtorzyłam ...`. Skąd się wzięła tablica w `getLastItem`?